.github/workflows/_changelog-entry-check.yml

```diff
@@ -1,4 +1,5 @@
-name: "Changelog entry check"
+name: "# Changelog entry check"
```

Comment:
What's the significance of the `#`?

Reply:
It's alphabetically after letters

Reply:
As it turns out, GitHub does not care about this and they still show up on top. I guess all we have is pinning.